New per task metrics based on sched debug. (#533)
intel/workload-collocation-agent: docs/api.rst

    If string is provided it will be used as regexp to match key.

- ``sched``: **Union[Str, bool]** = *False*

    Responsible for collecting data from /proc/PID/sched metric:
    - task_sched_stat (lines with ':'),
    - task_sched_stat_numa_faults (numa_faults field).

>     By default sched is enabled and all metrics (lines from /proc/PID/sched containg ':')

Author: ```suggestion
    By default "sched stats" are disabled no metrics.
```

>     will be collected.  False means disable the collection.

Author: ```suggestion
    will be collected.  True enables metrics collection.
```
